clang/test/CodeGenSYCL/address-space-swap.cpp

```diff
@@ -3,8 +3,8 @@
 
 
 void test() {
-  static int foo = 0x42;
-// CHECK: @[[FOO:[a-zA-Z0-9_]+]] = internal addrspace(1) global i32 66, align 4
+ int foo = 0x42;
```

Comment:
Alignment is off.

Reply:
I will fix it.

Reply:
We use LLVM coding style which assumes two-space indentation.

Reply:
Got it